webpack.config.js

```diff
@@ -6,7 +6,7 @@ var path = require('path');
 //                                               #idcommit
 
 module.exports = {
-  entry: path.resolve(__dirname, 'src/Main.js'),
+  entry: ['whatwg-fetch', path.resolve(__dirname, 'src/Main.js')],
```

Comment:
Not sure it's a good idea to "bundle" the polyfills; better declare them as [`externals`](https://webpack.github.io/docs/configuration.html#externals) IMO, and ask users to load them in addition to `itowns2.js` if they need it (want to support browsers with no native support).
We could revisit this later though (because the same reasoning could apply to other dependencies, such as Three.js).